sdk/servicebus/azure-servicebus/azure/servicebus/management/_generated/_configuration.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+
+from azure.core.configuration import Configuration
+from azure.core.pipeline import policies
+
+from ._version import VERSION
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Optional
+
+
+class ServiceBusManagementClientConfiguration(Configuration):
+    """Configuration for ServiceBusManagementClient.
+
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param endpoint: The Service Bus fully qualified domain name.
+    :type endpoint: str
+    """
+
+    def __init__(
+        self,
+        endpoint,  # type: str
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        if endpoint is None:
+            raise ValueError("Parameter 'endpoint' must not be None.")
+        super(ServiceBusManagementClientConfiguration, self).__init__(**kwargs)
+
+        self.endpoint = endpoint
+        kwargs.setdefault('sdk_moniker', 'servicebusmanagementclient/{}'.format(VERSION))
+        self._configure(**kwargs)
+
+    def _configure(
+        self,
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        self.user_agent_policy = kwargs.get('user_agent_policy') or policies.UserAgentPolicy(**kwargs)
```

Comment:
This is probably a question born out of my ignorance of autorest; but does it not generate docstrings for the actual kwargs it looks for?  Is this not expected to ever be consumed by a human?

Reply:
I don't think so. What I learnt today was azure-core is going to write documents. 
Now we only have this [CLIENT_LIBRARY_DEVELOPER.md](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md).
But this is for SDK developer instead of SDK users.